.spi.yml

```diff
@@ -1,4 +1,4 @@
 version: 1
 builder:
   configs:
-    - documentation_targets: [HealthCardControl, HealthCardAccess, NFCCardReaderProvider, CardReaderProviderApi]
\ No newline at end of file
+    - documentation_targets: [HealthCardAccess, NFCCardReaderProvider, CardReaderProviderApi]
```

Comment:
I removed this target since docc generation is broken here and probably hinders the documentation of the other targets.